lib/Github/Api/Integrations.php

```diff
@@ -26,4 +26,52 @@ public function createInstallationToken($installationId, $userId = null)
 
         return $this->post('/installations/'.rawurlencode($installationId).'/access_tokens', $parameters);
     }
+
+    /**
+     * Find all installations for the authenticated integration.
+     *
+     * @return array
+     */
+    public function findInstallations()
+    {
+        return $this->get('/integration/installations');
+    }
+
+    /**
+     * List repositories that are accessible to the authenticated installation.
+     *
```

Comment:
Add link to https://developer.github.com/v3/integrations/installations/#list-repositories